### PR TITLE
Prepare release 3.6

### PR DESCRIPTION
**Pull request by lldelisle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,7 +135,8 @@ Citation
 ---------
 If you use pyGenomeTracks in your analysis, you can cite the following paper :
 
-Fidel Ramírez, Vivek Bhardwaj, Laura Arrigoni, Kin Chung Lam, Björn A. Grüning, José Villaveces, Bianca Habermann, Asifa Akhtar & Thomas Manke. High-resolution TADs reveal DNA sequences underlying genome organization in flies. Nature Communications (2018) [doi:10.1038/s41467-017-02525-w](https://www.nature.com/articles/s41467-017-02525-w)
+Lopez-Delisle L, Rabbani L, Wolff J, Bhardwaj V, Backofen R, Grüning B, Ramírez F, Manke T. pyGenomeTracks: reproducible plots for multivariate genomic data sets. Bioinformatics. 2020 Aug 3:btaa692. [doi: 10.1093/bioinformatics/btaa692](https://doi.org/10.1093/bioinformatics/btaa692). Epub ahead of print. PMID: 32745185.
+
```
Comment (joachimwolff) [inline README.md:139]: I would not replace the old citation. Just mention both.

File: docs/content/citation.rst [hunk 2]
```diff
@@ -3,4 +3,4 @@ Citation
 
 If you use pyGenomeTracks in your analysis, you can cite the following paper :
 
-Fidel Ramírez, Vivek Bhardwaj, Laura Arrigoni, Kin Chung Lam, Björn A. Grüning, José Villaveces, Bianca Habermann, Asifa Akhtar & Thomas Manke. High-resolution TADs reveal DNA sequences underlying genome organization in flies. Nature Communications (2018) `doi:10.1038/s41467-017-02525-w <https://www.nature.com/articles/s41467-017-02525-w>`_.
+Lopez-Delisle L, Rabbani L, Wolff J, Bhardwaj V, Backofen R, Grüning B, Ramírez F, Manke T. pyGenomeTracks: reproducible plots for multivariate genomic data sets. Bioinformatics. 2020 Aug 3:btaa692. `doi: 10.1093/bioinformatics/btaa692 <https://doi.org/10.1093/bioinformatics/btaa692>`_. Epub ahead of print. PMID: 32745185.
```
Comment (joachimwolff) [inline docs/content/citation.rst:6]: Same here with the citation.

File: docs/content/releases/3.5.2.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
+3.5.2
+=====
+
+Python dependency with conda:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Using conda which is the way we recommand installation it is now possible to use python 3.8.
```
Comment (joachimwolff) [inline docs/content/releases/3.5.2.rst:7]: Upper case Python? Not sure actually

File: docs/content/releases/3.5.2.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+Relax version control on matplotlib:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Any version of matplotlib from 3.1.1 to 3.3.2 is now accepted.
```
Comment (joachimwolff) [inline docs/content/releases/3.5.2.rst:12]: In the other PR you limited matplotlib to >=3.1.1 and <= 3.2.2 ?

File: setup.py [hunk 5]
```diff
@@ -106,7 +106,7 @@ def checkProgramIsInstalled(self, program, args, where_to_download,
 setup(
     name='pyGenomeTracks',
     version=get_version(),
-    author='Fidel Ramírez, Vivek Bhardwaj, Joachim Wolf, Björn Grüning',
+    author='Lucille Lopez-Delisle, Leily Rabbani, Joachim Wolf, Björn Grüning',
     author_email='deeptools@googlegroups.com',
```
Comment (joachimwolff) [inline setup.py:110]: is anyone taking care of this google group? If not we should remove the reference to it in all documents.

Comment (lldelisle) [reply setup.py:110]: Perfect I am not part of it...

Comment (lldelisle) [reply setup.py:110]: What should I put instead? Can Freiburg uni provide us a mailing list? (For info it is also on hicexplorer)

File: README.md (hunk 1 above)
Comment (lldelisle) [reply README.md:139]: ok

Comment (lldelisle) [reply README.md:139]: So I ask user to cite both? (Personally I only cited the last one in my last publication but it is fine to put both in the next one).

File: docs/content/releases/3.5.2.rst (hunk 3 above)
Comment (lldelisle) [reply docs/content/releases/3.5.2.rst]: Yes you are right

File: docs/content/releases/3.5.2.rst (hunk 4 above)
Comment (lldelisle) [reply docs/content/releases/3.5.2.rst:12]: I made a mistake in the installation documentation, sorry. Thanks for the catch. I confirm 3.3.2.

File: README.md (hunk 1 above)
Comment (joachimwolff) [reply README.md:139]: Yes, we ask them to cite both; if they do it, is something else.

File: setup.py (hunk 5 above)
Comment (joachimwolff) [reply setup.py:110]: We simply don't provide a mail?

File: README.md (hunk 1 above)
Comment (lldelisle) [reply README.md:141]: ok done

File: docs/content/citation.rst (hunk 2 above)
Comment (lldelisle) [reply docs/content/citation.rst:8]: also done

File: setup.py (hunk 5 above)
Comment (lldelisle) [reply setup.py]: ok done

**Review by joachimwolff — APPROVED** [review]

**Pull request merged**